.github/workflows/release.yml

```diff
@@ -20,6 +25,4 @@ jobs:
           cache: "npm"
 
       - run: npm ci
-      - run: npm publish --access public
```

Comment:
The `public` access only needs to be specified on the first publication, it is redundant afterwards.